snap/snapcraft_strictmode.yaml

```diff
@@ -0,0 +1,69 @@
+name: sosreport
+summary: Sos is an extensible, portable, support data collection tool
+description: |
+  Sos is an extensible, portable, support data collection tool
+  primarily aimed at Linux distributions and other UNIX-like operating
+  systems.
+grade: stable
+base: core22
+confinement: strict
+adopt-info: sos
+license: GPL-2.0-or-later
+environment:
+  PYTHONPATH: ${SNAP}/lib/python3.10/site-packages:${SNAP}/usr/lib/python3/dist-packages:${PYTHONPATH}
+  MAGIC: ${SNAP}/usr/share/file/magic.mgc
+
+parts:
+  sos:
+    plugin: python
+    source: .
+    override-pull: |
+      craftctl default
+      craftctl set version="$(git describe --tags --always)"
+    build-attributes:
+      - enable-patchelf
+    build-packages:
+      - git
+      - python3
+      - snapcraft
+      - gettext
+      - python3-venv
+    stage-packages:
+      - coreutils
+      - cloud-init
+      - cryptsetup-bin
+      - e2fsprogs
+      - fdisk
+      - iproute2
+      - libatm1
+      - libpam-modules-bin
+      - mount
+      - procps
+      - systemd
+      - udev
+      - util-linux
+      - python3-magic
+      - python3-packaging
+      - python3-pexpect
+  cleanup:
+    after: [sos]
+    plugin: nil
+    build-snaps: [core22]
+    override-prime: |
+      set -eux
+      for snap in "core22"; do  # List all content-snaps you're using here
```

Comment:
You plan to extend the list of snaps in future, right? As currently the for cycle with 1 iteration makes a little sense..

Reply:
good point, it was a copy and paste from somewhere else. I will solve that, once I've figured out why the stageone tasks are not going through, and solve these.